consensus/polybft/state_stats.go

```diff
@@ -154,3 +158,44 @@ func (s *State) startStatsReleasing() {
 		prev = stats
 	}
 }
+
+func polybftMetrics(rootnodeURL string,
+	validatorAddress ethgo.Address,
+	closeCh <-chan struct{},
+	logger hclog.Logger,
+	interval time.Duration) {
+	// zero means metrics are disabled
+	if interval <= 0 {
+		return
+	}
+
+	gweiPerWei := new(big.Int).Exp(big.NewInt(10), big.NewInt(9), nil) // 10^9
+
+	ticker := time.NewTicker(interval)
+	defer ticker.Stop()
+
+	rpcClient, err := jsonrpc.NewClient(rootnodeURL)
+	if err != nil {
+		logger.Error("metrics - connection to root node failed", "err", err)
+
+		return
+	}
+
+	for {
+		select {
+		case <-closeCh:
+			return
+		case <-ticker.C:
+			balance, err := rpcClient.Eth().GetBalance(validatorAddress, ethgo.Latest)
+			if err != nil {
+				logger.Error("metrics get balance call failed", "err", err)
+
+				continue
+			}
+
+			balanceInGwei := new(big.Int).Div(balance, gweiPerWei).Uint64()
```

Comment:
Why not floating point division by relying on `big.Float`s?

Reply:
Fixed https://github.com/0xPolygon/polygon-edge/pull/1941/commits/61e9e8d0044a2482ee232649943a7e9861e56c58